Error compiling with React Native 0.57.0

After updating to the last version of React Native (0.57.0) I can not compile and build the APK because of the error below:

```
:react-native-carrier-info:verifyReleaseResources
error: resource android:style/TextAppearance.Material.Widget.Button.Borderless.Colored not found.
error: resource android:style/TextAppearance.Material.Widget.Button.Colored not found.
/root/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/1f4e1fccb69ec83f875c934344078536/res/values-v26/values-v26.xml:9:5-12:13: AAPT: error: resource android:attr/colorError not found.
    
/root/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/1f4e1fccb69ec83f875c934344078536/res/values-v26/values-v26.xml:13:5-16:13: AAPT: error: resource android:attr/colorError not found.
    
/root/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/1f4e1fccb69ec83f875c934344078536/res/values-v26/values-v26.xml:17:5-93: AAPT: error: style attribute 'android:attr/keyboardNavigationCluster' not found.
    
/root/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/1f4e1fccb69ec83f875c934344078536/res/values/values.xml:251:5-69: AAPT: error: resource android:attr/fontStyle not found.
    
/root/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/1f4e1fccb69ec83f875c934344078536/res/values/values.xml:251:5-69: AAPT: error: resource android:attr/font not found.
    
/root/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/1f4e1fccb69ec83f875c934344078536/res/values/values.xml:251:5-69: AAPT: error: resource android:attr/fontWeight not found.
    
error: failed linking references.
:react-native-carrier-info:verifyReleaseResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-carrier-info:verifyReleaseResources'.
> com.android.ide.common.process.ProcessException: Failed to execute aapt
```

Any solution for that?

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -1,17 +1,20 @@
 apply plugin: 'com.android.library'
 
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion safeExtGet('compileSdkVersion', 28)
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 22
+        minSdkVersion safeExtGet('minSdkVersion', 16)
+        targetSdkVersion safeExtGet('targetSdkVersion', 28)
         versionCode 1
         versionName "1.0"
     }
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
+    implementation "com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}"
 }

